### DISCUSSION TITLE
Player Widget show master rank instead of hunter rank

### DISCUSSION THREAD

**Discussion by thieleju** [HunterPie/HunterPie#478] [Q&A] [answered]
Since version 1 showed both, master and hunter rank of all players, I was wondering if there is a setting to switch to only showing master rank/showing both in the player widget.

**Comment by haato3o** [comment, marked as answer]
The reason why I made it show only the MR is due to the fact that HR isn't really that relevant in the end game, but I can add an option to let you pick which one to pick, or to show both.

**Reply by thieleju** [reply]
Oh I thought it only showed HR, only MR is perfect
